### DISCUSSION TITLE
How to change the email for acme

### DISCUSSION THREAD

**Discussion by leic4u** [0xJacky/nginx-ui#208] [Q&A] [answered]
I noticed that issuing a cert with the email which is filled in the nginx-ui install page, but I didn't find how to change it.

Anyone knows that how could I change the email?

**Comment by 0xJacky** [comment, marked as answer]
You can change it in `/usr/local/etc/nginx-ui/app.ini` if you used [installation scripts for Linux](https://github.com/0xJacky/nginx-ui#script-for-linux).

**Reply by leic4u** [reply]
Thanks, I install with docker, and I find it in `./nginx-ui/app.ini`.

How about develop a feature to modify the `app.ini` in web UI? @0xJacky
